src/main/resources/application.yaml

```diff
@@ -20,6 +20,7 @@ azure:
     envelopes:
       connection-string: ${ENVELOPES_QUEUE_CONNECTION_STRING}
       queue-name: envelopes
+      max-delivery-count: ${ENVELOPES_QUEUE_MAX_DELIVERY_COUNT:10}
```

Comment:
It would be best to have no default value, so that if we forget to configure it in any environment, the app won't even start there.